CHANGELOG.md

```diff
@@ -5,9 +5,11 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 - `propagate_flags` keyword for `UVData.frequency_average` which flags averaged samples if any contributing samples were flagged
+- `nsample_array_dtype` keyword for `UVData.read`, which sets the datatype for the nsample_array. Currently only used for mwa_corr_fits files.
```

Comment:
This should be at the top -- reverse chronological order.

Reply:
Good catch, thanks! Should be fixed now.